src/api/ci/jenkins/utils/jenkins.utils.ts

```
@@ -0,0 +1,264 @@
+      .map(segment => `job/${encodeURIComponent(segment)}`)
+      .join('/');
+  }
+
+  /**
+   * Build build-specific API path
+   */
+  static buildBuildPath(jobName: string, buildNumber: number, folderName?: string, endpoint: string = ''): string {
+    const jobPath = this.buildJobPath(jobName, folderName);
+    return endpoint ? `${jobPath}/${buildNumber}/${endpoint}` : `${jobPath}/${buildNumber}`;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)